### ISSUE TITLE
OpenTTD version error strings mention wrong JGRPP version

### ISSUE THREAD

**Issue by James103** [2TallTyler/improved_town_industries#113]
Currently, all of the version error strings look like this:
```
Improved Town Industries requires OpenTTD version 14.0, JGR version 0.85, or better
```

This should be fixed to say:
```
Improved Town Industries requires OpenTTD version 14.0, JGR version 0.58, or better
```

(note `0.85` changed to `0.58`)

Caused by https://github.com/2TallTyler/improved_town_industries/pull/112.

**Comment by 2TallTyler** [comment]
Oops, seems I transposed the numbers.

**Comment by 2TallTyler** [comment]
Fixed in bdcd90d.
